### PR TITLE
(I hate this) Weeks are based on txt files

### PR DESCRIPTION
**Pull request by Lucky-56**
This is basically a 'make easier to comprehend' rather than a 'make easier to mod'.
Basically useless, but if you want to merge that's on you. 😉

### CODE REVIEW

**Comment by ACardboardBox1** [comment]
soon the whole game will be txt files

**Review by puyoxyz — CHANGES_REQUESTED** [review]
why are you separating them into THREE DIFFERENT FILES

(also again there is no point to this because you need to recompile anyways if you add new songs)

File: source/StoryMenuState.hx [hunk 1]
```diff
@@ -72,8 +49,37 @@ class StoryMenuState extends MusicBeatState
...
+		var swagGoodSecondArray:Array<Array<String>> = [];
+		var swagGoodThirdArray:Array<Array<String>> = [];
+
+		for (i in 0...initList.length)
+		{
+			var data:Array<String> = initList[i].split('; ');
+
+			for(i in 0...data.length)
+			{
+				var dataTwo:Array<String> = data[i].split(', ');
```
Comment (puyoxyz) [inline source/StoryMenuState.hx]: don't force the space after `;`/`,`. instead, do `split(',')`/`split(';')` then [trim()](https://api.haxe.org/StringTools.html#trim) it

**Review by puyoxyz — CHANGES_REQUESTED** [review]

File: source/StoryMenuState.hx (hunk 1 above)
Comment (Lucky-56) [reply source/StoryMenuState.hx]: okay

**Review by puyoxyz — COMMENTED** [review]
seems cool but needs to be tested (cant myself, kinda busy with stuff right now;; if noone tests and reviews in like a day or two request my review again)

but again this is useless: 
> there is no point to this because you need to recompile anyways if you add new songs

**Comment by Lucky-56** [comment]
hey @puyoxyz, fun fact this fixes my problem I'm having with [patch-3](https://github.com/KadeDev/Kade-Engine/pull/894#discussion_r654770492),
so not really useless, but I'm sure there's another way to fix the issue on [patch-3](https://github.com/KadeDev/Kade-Engine/pull/894#discussion_r654770492)

EDIT: yes there was

**Comment by Lucky-56** [comment]
how about I change it that instead of it being a txt file this just puts all of the week data into one array ?

**Comment by Lucky-56** [comment]
nvm, I just noticed that I hate this
## Pick it up if you really want to (but I doubt)

**Pull request closed without merging**